public/js/stores/ArticleStore.js

```diff
@@ -5,18 +5,34 @@ class ArticleStore extends EventEmitter {
     constructor() {
         super()
         this.articles = []
+        this.sort = 'NEW'
```

Comment:
I think new is better unless we can think of a way to bias the top articles on the front page. If its top then the top ones will always get more views therefore more votes. We can do what reddit does and go with "HOT" which is basically recent top articles but thats another feature